internal/core/application/wallet_service.go

```diff
@@ -400,16 +412,28 @@ func sendToMany(opts sendToManyOpts) (string, error) {
 		return "", err
 	}
 
+	for _, v := range feeUpdateResult.SelectedUnspents {
+		inputBlindingData[index] = wallet.BlindingData{
+			Asset:         v.Asset(),
+			Amount:        v.Value(),
+			AssetBlinder:  v.AssetBlinder(),
+			AmountBlinder: v.ValueBlinder(),
+		}
```

Comment:
forgot to increment `index`?

Reply:
no its in L393

Reply:
That's done when looping over `updateResult.SelectedUnspents`, here we're looping over `feeUpdateResult.SelectedUnspents`!

Reply:
cc @sekulicd

Reply:
i expected that feeUpdateResult could add only one more input for fee, if this is true that no need for increment.
@altafan can we have more than one input?

Reply:
Of course it is possible to use more than one unspents to pay for fees.. as this is done, it would give problems for sure in such cases. Please fix it.

Reply:
Done